src/Inventory/Models/BaseModel.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+namespace Kanvas\Inventory\Models;
+
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model as EloquentModel;
+use Kanvas\Enums\StateEnums;
+use Kanvas\Traits\SoftDeletes;
+
+class BaseModel extends EloquentModel
+{
+    use HasFactory;
+    //use SoftDeletes;
+
+    protected $attributes = [
+        'is_deleted' => 0,
+    ];
+
+    /**
+     * Get by uui.
+     *
+     * @param string $uuid
+     *
+     * @return self
+     */
+    public static function getByUuid(string $uuid) : self
```

Comment:
@FredPeal suggestion move this 2 methods to a trait so you can reuse on other models in the future